src/NodaTime.Demo/LocalDateDemo.cs

```diff
@@ -35,6 +35,14 @@ public void CombineWithTime()
             Assert.AreEqual(new LocalDateTime(2010, 6, 16, 16, 20, 0), dateTime);
         }
 
+        [Test]
+        public void Add()
+        {
+            LocalDate date = new LocalDate(2010, 6, 16);
+            LocalDate dateTime = Snippet.For(LocalDate.Add(date, Period.FromDays(3)));
```

Comment:
This isn't a `dateTime` - could you rename the variables to maybe `startDate` and `result`?